schema/deploy/warehouse/identifier/triggers/barcode-distance-check.sql

```diff
@@ -57,7 +57,8 @@ create function warehouse.identifier_barcode_distance_check() returns trigger as
             return NEW;
         end if;
     end
-$$ language plpgsql;
+$$ language plpgsql
+   security definer;
```

Comment:
I think you need to do a `sqitch rework` to drop and re-create the function/triggers with the security definer rather than edit the original function

Reply:
Jover's correct: the way to update this function is to do a `sqitch rework` of the change.